CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ Current
 -------
 
 ### Fixed:
+- [Fix contains filter behavior](https://github.com/yahoo/fili/pull/998)
+    * When there is a Contains filter applied to a non-cached dimension, it will be translated into a `SeachFilter` instead of `SelectorFilter`.
```

Comment:
```suggestion
    * When there is a Contains filter applied to a non-cached dimension, it will be translated into a `SearchFilter` instead of `SelectorFilter`.
```